lib/commands/execute.js

```diff
@@ -10,6 +10,8 @@ const WINDOWS_EXTENSION_SCRIPT_PATTERN = /^windows:/;
 const EXTENSION_COMMANDS_MAPPING = {
   startRecordingScreen: 'startRecordingScreen',
   stopRecordingScreen: 'stopRecordingScreen',
+  deleteFile: 'windowsDeleteFile',
```

Comment:
I'm still not quite convinced about having these two extensions and pushFile call since they could be used to perform a potential attack on the system. @ggaller What do you think about putting them under a security feature, like we did for [PowerShell calls](https://github.com/appium/appium-windows-driver/blob/80d84ec6322e5681207efac54e73c0268c008df8/lib/commands/execute.js#L21) ?

cc @jlipps @KazuCocoa

Reply:
I thought about security issues and saw how the security feature works in PowerShell. Maybe you are right, and all file manipulation commands need to be putting under the security flag. I also thought about limiting the available folders, which is easy to do for UWP apps (`%LocalAppData%\Packages\<PackageFamilyName>\`), but more difficult for classic apps.

Reply:
a security extension for this seems reasonable.

Reply:
@mykola-mokhnach any idea how run e2e tests with security flags? When i running driver as a server it not parse any arguments from command line.

Reply:
Added `delete_files` security feature for `deleteFile, deleteFolder` commands.

Reply:
Yes, you could do something like https://github.com/appium/appium-mac2-driver/blob/c474f7e3cfc198480a78f6fcc282b0ffc77cbf85/lib/server.js#L7

Reply:
Thanks. Added `relaxedSecurityEnabled` argument to `startServer` method.